presto-docs/src/main/sphinx/connector/alluxio.rst

```diff
@@ -0,0 +1,99 @@
+**Overview**
+
+`Alluxio <www.alluxio.io>`_ is a data orchestration  open source distributed virtual file system which provides a block-level read/write caching engine for Presto connecting to a variety of disparate storage systems including S3 and HDFS. Presto can query files stored in Alluxio through the Hive connector. 
+Basic Configuration
+
+Presto can use the Hive Metastore for the database and table metadata information, as well as the file system location of table data. In order to enable this, create and edit the Presto configuration ``etc/catalog/hive.properties``:
+
+``connector.name=hive-hadoop2``
+``hive.metastore.uri=thrift://localhost:9083``
+
+Copy Alluxio client jar ``/<ALLUXIO_HOME>/client/alluxio-<VERSION>-client.jar`` into directory ``plugin/hive-hadoop2/`` on all Presto servers.
+Additional Alluxio Configuration
```

Comment:
Would it make sense to add a link to the Alluxio configuration options?

Reply:
Thanks for the feedback. Made it more modular and added some links.